sns.tf

```diff
@@ -0,0 +1,42 @@
+data "aws_iam_policy_document" "sns_for_github" {
+
+  statement {
+    sid    = "AllowListDescribe"
+    effect = "Allow"
+    actions = [
+      "sns:ListPlatformApplications",
+      "sns:ListSubscriptions",
+      "sns:ListTagsForResource",
+      "sns:ListTopics"
+    ]
+    resources = ["*"]
+  }
+
+  statement {
+    sid    = "AllowPublishOwn"
+    effect = "Allow"
+    actions = [
+      "sns:Publish"
```

Comment:
I'd also add:
- `GetPlatformApplicationAttributes` to complement `sns:ListPlatformApplications` above
- `GetSubscriptionAttributes` to complement `sns:ListSubscriptions` above
- `GetTopicAttributes`